### PR TITLE
Docker Support

### PR DESCRIPTION
**Pull request by ericjohnson97**
This PR runtime docker support

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Build
```
Comment (ericjohnson97) [inline Dockerfile:1]: @patrickelectric I am not sure if you would like to have a separate build container for building or if you prefer it like this. let me know and I can adjust it.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+# Runtime environment
+FROM alpine
+WORKDIR /root/
+
+ARG TARGET_ARCH=x86_64-unknown-linux-musl
+
+COPY --from=builder /usr/src/mavlink2rest/target/${TARGET_ARCH}/release/mavlink2rest ./mavlink2rest
+
+ENV MAVLINK_SRC="udpin:127.0.0.1:14551"
```
Comment (ericjohnson97) [inline Dockerfile:22]: Just realized i forgot to make the other options ENV variables. on my next commit I can add support for the other options

Comment (patrickelectric) [reply Dockerfile:22]: Thanks!

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ENV SERVER_PORT="0.0.0.0:8088"
+ENV RUST_BACKTRACE=1
+
+RUN chmod +x mavlink2rest
+
+ENTRYPOINT ./mavlink2rest -c ${MAVLINK_SRC} -s ${SERVER_PORT}
```
Comment (patrickelectric) [inline Dockerfile:28]: Can we add an env variable for generic arguments ?

Comment (ericjohnson97) [reply Dockerfile]: let me know what you think of the revised args

**Comment by ericjohnson97** [comment]
@patrickelectric Let me know what you think of this dockerfile. If this is not what you want or would like to do it a different way, I understand. I can either revise or let you do it in some time in the future.

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**